### PR TITLE
Remove the `FIR` prefix from notification constants.

### PR DESCRIPTION
**Pull request by ryanwilson**

### CODE REVIEW

File: FirebaseInstallations/Source/Library/Public/FirebaseInstallations/FIRInstallations.h [hunk 1]
```diff
@@ -22,9 +22,11 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /** A notification with this name is sent each time an installation is created or deleted. */
-FOUNDATION_EXPORT const NSNotificationName FIRInstallationIDDidChangeNotification;
+FOUNDATION_EXPORT const NSNotificationName FIRInstallationIDDidChangeNotification
+    NS_SWIFT_NAME(InstallationIDDidChange);
```
Comment (ryanwilson) [inline FirebaseInstallations/Source/Library/Public/FirebaseInstallations/FIRInstallations.h:26]: The `Notification` was already dropped here in Swift, the only difference is a lack of `FIR` prefix now.

**Comment by ryanwilson** [comment]
cc @chliangGoogle for the Messaging sample change

**Review by maksymmalyhin — APPROVED** [review]
Thank you for the improvements!

**Review by chliangGoogle — APPROVED** [review]
LGTM on sample app!

**Comment by ryanwilson** [comment]
Thanks! Fix for the FIS failure is at https://github.com/firebase/quickstart-ios/pull/1055

**Review by paulb777 — APPROVED** [review]

**Pull request merged**